src/util/customTransformer.js

```diff
@@ -104,11 +108,6 @@ async function runUserTransform(
     }),
   );
 
-  await jail.set('_rsSecrets', function (...args) {
-    if (args.length == 0 || !secrets || !secrets[args[0]]) return 'ERROR';
```

Comment:
passing secrets in `runUserTransform` can be removed too?

Reply:
https://github.com/rudderlabs/rudder-transformer/blob/2a6e2b4168384817470dccb8923d684df33b28af/src/util/customTransformer.js#L18